src/services/job_service.py

```diff
@@ -73,29 +73,22 @@ def schedule_cron_job(job_scheduler: BackgroundScheduler, function: Callable, jo
         raise ValueError(f"Failed to schedule cron job '{job.job_uid}'. {e}'")
 
 
-def _get_job(job_uid: UUID) -> Union[Job, None]:
+def _get_job(job_uid: UUID) -> Job:
     """Get a job from the job storage."""
     try:
         if raw_job := get_job_store().get(str(job_uid)):
-            return Job.from_dict(json.loads(raw_job.decode()))
+            return Job(**json.loads(raw_job.decode()))
+        else:
+            raise NotFoundException(f"No job with id '{job_uid}' is registered")
     except AuthenticationError:
         raise ValueError(
             "Tried to fetch a job from Redis but no password"
             + " was supplied. Make sure SCHEDULER_REDIS_PASSWORD is set."
         )
-    return None
 
 
 def _set_job(job: Job):
-    return get_job_store().set(str(job.job_uid), json.dumps(job.to_dict()))
-
-
-def load_cron_jobs():
-    for key in get_job_store().scan_iter():
-        job = _get_job(UUID(key.decode()))
-        if job.cron_job:
-            schedule_cron_job(scheduler, _run_job, job)
-            logger.info(f"Loaded and registered job '{job.job_uid}' from {config.SCHEDULER_REDIS_HOST}")
```

Comment:
This should still be in use. 
We need a way to re-register cron-jobs after restarting/deploying a new container

Reply:
ok. I noticed it was not ues anywhere so i assumed it was old :)